lib/path.js

```diff
@@ -1,10 +1,10 @@
 'use strict';
 
-const inspect = require('util').inspect;
+const errors = require('internal/errors');
 
 function assertPath(path) {
   if (typeof path !== 'string') {
-    throw new TypeError('Path must be a string. Received ' + inspect(path));
+    throw new errors.TypeError('ERR_INVALID_ARG_TYPE', 'path', 'String');
```

Comment:
The `typeof path` was not passed before the change. The `inspect(path)` returned the value, this is not something the `ERR_INVALID_ARG_TYPE` supports.